bigbluebutton-html5/imports/ui/components/media/styles.scss

```diff
@@ -96,7 +96,15 @@ $after-content-order: 3;
 }
 
 .hideOverlay {
+  visibility: hidden !important;
```

Comment:
Didn't you just remove these a few PRs ago?!

Reply:
Yeah, I'm added them back because of an undetected bug that it caused. (without them the presentation area can't be fully minimized)